xwrf/config.yaml

```
@@ -1,5 +1,30 @@
 version: 0.1
 
+latitude_coords:
+  - CLAT
+  - XLAT
+  - XLAT_C
+  - XLAT_M
+  - XLAT_U
+  - XLAT_V
+
+longitude_coords:
+  - CLONG
+  - XLONG
+  - XLONG_C
+  - XLONG_M
+  - XLONG_U
+  - XLONG_V
+
+time_coords:
+  - XTIME
+  - Times
+  - Time
+  - time
+boolean_units_attrs:
```

```suggestion
  - time

boolean_units_attrs:
```

Definitely not a needed change, but the inconsistent line breaks stood out.